core/src/main/java/io/grpc/util/RoundRobinLoadBalancer.java

```diff
@@ -126,6 +133,8 @@ public void onSubchannelState(ConnectivityStateInfo state) {
     for (Subchannel removedSubchannel : removedSubchannels) {
       shutdownSubchannel(removedSubchannel);
     }
+
+    return !resolvedAddresses.getAddresses().isEmpty();
```

Comment:
This is guaranteed to return `true`, right, because of the check at the beginning of the method?